src/Tickets/Commerce/Gateways/PayPal/Gateway.php

```diff
@@ -139,13 +143,16 @@ public static function is_enabled() {
 		}
 
 		// If option is not explicitly set, the default will be if PayPal is connected.
-		return self::is_connected();
+		return static::is_connected();
 	}
 
 	/**
 	 * @inheritDoc
 	 */
-	public static function get_checkout_template_vars() {
-		return tribe( Buttons::class )->get_checkout_template_vars();
+	public function render_checkout_template( \Tribe__Template $template ): string {
```

Comment:
Does the return a string or simply render HTML?

Reply:
That is actually a good point, I like to always return the value even if I am echoing it to the screen.